lib/nostrum/struct/message.ex

```diff
@@ -104,6 +151,10 @@ defmodule Nostrum.Struct.Message do
   """
   @type application :: Application.t() | nil
 
+  @typedoc """
+  if the message is a response to an Interaction, this is the id of the interaction's application
```

Comment:
This should match the format of the other docstrings.

```suggestion
  If the message is a response to an interaction, this is the ID of the interaction's application
```